src/main/GraphReader.java

```diff
@@ -0,0 +1,60 @@
+package main;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+/**
+ * Created by lelay on 08.10.17.
+ */
+public class GraphReader {
+
+    private String path;
+
+    GraphReader(String[] args){
+        /*command line arguments:
+        1) the first is the path to a file
+        */
+        this.path = args[0];
+    }
+
+    public String getPath(){
+        return this.path;
+    }
+
+    public void setPath(String path){
+        this.path = path;
+    }
+
+    //isBiDirectional parameter means if it's a bi-directional graph or not
+    public int[][] readFile(boolean isBiDirectional) throws FileNotFoundException {
```

Comment:
@greenbarrow I actually used matrix form like you said 2 days ago that "it'll be better to use           
2-dimensional massive instead of matrix class", so I have to do this

Reply:
@greenbarrow I agree with you that isBiDirectional paramether is not rly beautiful realisation, but it works so... why not? :wink: